skyportal/handlers/api/gcn.py

```diff
@@ -261,6 +261,15 @@ def post_skymap_from_notice(dateobs, notice_id, user_id, session, asynchronous=T
         session.commit()
         localization_id = localization.id
 
+        if url is not None:
+            r = requests.get(url, allow_redirects=True)
+            data_to_disk = r.content
+            urlpath = urlsplit(url).path
+            localization_name = os.path.basename(urlpath)
+            if data_to_disk is not None:
+                localization.save_data(localization_name, data_to_disk)
+                session.commit()
```

Comment:
File availability from Fermi or LVC has been a little random. So please make sure that this runs last after we ingest everything else, and wrap the requests.get in a try except so we avoid any problems.